.github/workflows/test-suite.yml

```diff
@@ -84,4 +84,42 @@ jobs:
     if: ${{ !cancelled() }}
     secrets: inherit
     uses: ./.github/workflows/cleanup-test-env.yml
-  
\ No newline at end of file
+  
+  slack-notification:
+    needs: [mig-tests, acc-tests, clean-after]
+    if: ${{ (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') && (needs.mig-tests.result == 'failure' || needs.acc-tests.result == 'failure')}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send Slack message
+        id: slack
+        uses: slackapi/slack-github-action@v1.25.0
+        with:
+          payload: |
+            {
+              "text": ":red_circle: Test Suite failed",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "*Test Suite failed* @apix-integrations-on-call"
```

Comment:
make `@apix-integrations-on-call` a secret. This is internal and should not be shown in a public repo

Reply:
Payload of the message can be seen in the [action](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/8388964622/job/22974572076) `Send Slack Message` step, so even if making this a secret it will be displayed. Open to suggestions to hide it but seems it's not possible with this action

Reply:
i'm not sure it will show up in the action,   GH removes all strings that match some secret

Reply:
Changed to use the secret! Thanks for all the inputs